libraries/AC_WPNav/AC_WPNav_OA.h

```diff
@@ -26,8 +26,9 @@ class AC_WPNav_OA : public AC_WPNav
     // Sets the waypoint destination using NEU coordinates in meters.
     // - destination_neu_m: NEU offset from EKF origin in meters.
     // - is_terrain_alt: true if the Z component represents altitude above terrain.
+    // arc_rad specifies the signed arc angle in radians for an ARC_WAYPOINT segment (0 for straight path)
     // - Resets OA state on success.
-    bool set_wp_destination_NEU_m(const Vector3p& destination_neu_m, bool is_terrain_alt = false) override;
+    bool set_wp_destination_NEU_m(const Vector3p& destination_neu_m, bool is_terrain_alt = false, float arc_rad = 0.0) override;
```

Comment:
I'm not really a fan of adding an optional parameter to these methods.  Optional parameters always seem to come back and bite you in the arse.

OTOH, it doesn't look like the alternatives would make this prettier, so all good.